packages/drivechain_client/README.md

```diff
@@ -0,0 +1,16 @@
+# drivechain_client
+
+A new Flutter project.
+
+## Getting Started
+
+This project is a starting point for a Flutter application.
+
+A few resources to get you started if this is your first Flutter project:
+
+- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
+- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
+
+For help getting started with Flutter development, view the
+[online documentation](https://docs.flutter.dev/), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
```

Comment:
Could we do a really simple non-boilerplate README?

```
This package contains a frontend for interacting with a Drivechain-enabled layer 1 bitcoin network.
```

And then maybe super simple instructions for starting the app in dev mode?